app/components/data-table-row-current.js

```diff
@@ -43,6 +44,21 @@ export default Component.extend({
     return false;
   }),
 
+  displayEmptyCell: computed('rowConfig', 'pop', function () {
```

Comment:
Can you explain what this property represents and why you need it? I'm having a hard time following the code.

Reply:
I will have to hold off on requesting a review. I need to make further changes to the PR.